doc/user/content/get-started/install-materialize-emulator.md

```diff
@@ -11,33 +11,27 @@ menu:
 
 The Materialize Emulator is an all-in-one Docker image available on Docker Hub
 for testing and evaluation purposes. The Materialize Emulator is not
-representative of Materialize's performance and full feature set. To evaluate
-Materialize for production scenarios, sign up for a [free trial
-account](https://materialize.com/register/?utm_campaign=General&utm_source=documentation)
-or [schedule a demo](https://materialize.com/demo/?utm_campaign=General&utm_source=documentation).
-
-{{< warning >}}
-
-The Materialize Emulator is for <redb> testing/evaluation purposes only. It is
-not suitable for production deployments</redb>.
-
-{{</ warning >}}
-
-### Materialize Emulator vs Materialize
-
-|                               | Materialize Emulator                                       | Materialize                                                            |
-|-------------------------------|----------------------------------------------------|------------------------------------------------------------------------|
-|         | <i>The fastest option for prototyping or internal testing with Materialize.</i> | <i>Cloud-hosted SaaS solution for critical operational workloads.</i>|
-| **Production deployments**  | ❌ Not suitable due to performance and [license limitations](#license-and-privacy-policy). | ✅ |
-| **Performance**             | ❌ Limited. Services are bundled in a single container. | ✅ High. Services are scaled across many machines.|
-| **Dedicated support**       | ❌ | ✅ |
-| **Sample data**  | ✅ Quickstart data source. | ✅ Quickstart data source. |
-| **Data sources**  | ✅ Connect using configuration SQL. | ✅ Connect using a streamlined GUI. |
-| **Version upgrades**                 | ✅ Manual, with no data persistence.| ✅ Automated, with data persistence. |
-| **Use case isolation**               | ❌ | ✅ |
-| **Fault tolerance**                  | ❌ | ✅ |
-| **Horizontal scalability**           | ❌ | ✅ |
-| **GUI (Materialize Console)**      | ✅ | ✅ |
+representative of Materialize's performance and full feature set.
+
+{{< important >}}
+
+The Materialize Emulator is <redb> not suitable production workloads.</redb>.
+
+{{</ important >}}
+
+### Materialize Emulator
+
+Materialize Emulator is easiest way to get started with Materialize, but is not
+suitable for full feature set evaluations or production workloads.
+
+| Materialize Emulator              | Details    |
+|-----------------------------------|------------|
+| **What is it**                    | A single Docker container version of Materialize. |
+| **Best For**                       | Very quick prototyping and CI jobs. |
```

Comment:
I wonder if "very quick" is redundant and should make it `Prototyping and CI jobs.`?

Reply:
Oh .. heh .. this mirrors our marketing blurbs (or at least, a version of it). But, will also double check in case things have changed since then (this was an old branch)

Reply:
Ah .. since it's just in the Raw copy tab of a spreadsheet, I think I can get rid of the Very quick.